internal/release/minisign.pub

```diff
@@ -0,0 +1,2 @@
+untrusted comment: minisign public key: 4C9786A7A6FF1D33
```

Comment:
This is the minisign public key used to sign the official bufbuild/plugins releases. Added it here so we don't have to try to parse it from release notes or anything like that.